src/components/organisms/EditProfileModal/EditProfileModal.tsx

```diff
@@ -74,15 +74,15 @@ const EditProfileModal: React.FunctionComponent<PropsType> = ({
               placeholder="Your display name"
               ref={register({
                 required: true,
-                maxLength: 16,
+                maxLength: 30,
```

Comment:
Instead of duplicating this number across the various locations in the code, it would be good to extract it into a `const` or 2 in `src/settings.ts`, then reuse it here.